config/dev.exs

```diff
@@ -20,6 +20,20 @@ config :phoenix_live_view, :debug_heex_annotations, true
 # Disable swoosh api client as it is only required for production adapters.
 config :swoosh, :api_client, false
 
+config :joken,
+  default_signer: [
+    signer_alg: "Ed25519",
+    key_openssh: """
+    -----BEGIN OPENSSH PRIVATE KEY-----
+    b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW
+    QyNTUxOQAAACDVgskcQdNGPgcP9UJIwA6AB1FUnvCyO19dChVY3EFuZQAAAKDVn3NU1Z9z
+    VAAAAAtzc2gtZWQyNTUxOQAAACDVgskcQdNGPgcP9UJIwA6AB1FUnvCyO19dChVY3EFuZQ
+    AAAECOw1cqNcGfb/U3HgERb+cujt5dvVM+QzIWMMEWeaua5NWCyRxB00Y+Bw/1QkjADoAH
+    UVSe8LI7X10KFVjcQW5lAAAAF2FyZW5hQGdhdGV3YXkubWlycmEuZGV2AQIDBAUG
+    -----END OPENSSH PRIVATE KEY-----
+    """
+  ]
+
```

Comment:
This will be changed in the future?

Reply:
Nope, why?

Reply:
Aahhh this is right since this is only being use for dev enviroment right? the real one comes from `System.get_env("JWT_PRIVATE_KEY")`

Reply:
Yep, this is only to simplify local so we don't to generate one

Reply:
Nice!